the-badass-todo-list-part-1/input.js

```diff
@@ -18,16 +20,20 @@
         };
         // To prevent user saving multiple times.
         input.disabled = true;
+        // Quiz #2
         saveData(data, tidyUp);
       }
     });
   }
 
+  // Quiz #2
```

Comment:
Yes we have two duplicated `saveData` in our program. But since we have closure, in theory you can always encapsulate them to avoid these issues. Of course, as what I've said, abusing closures can cause more issues than what it solves.